src/elasticsearch/mod.rs

```diff
@@ -132,7 +137,29 @@ impl Elasticsearch {
     pub fn client() -> &'static ureq::Agent {
         lazy_static::lazy_static! {
             static ref AGENT: ureq::Agent = {
-                ureq::AgentBuilder::new()
+                let agent_builder = ureq::AgentBuilder::new();
+
+                #[cfg(feature = "native_tls")] // if native_tls feature is enabled - we need to create TlsConnector and pass it to agent_builder
+                {
+                    match native_tls::TlsConnector::new() {
+                        Ok(tls_config) => {
+                            return agent_builder
+                            .tls_connector(Arc::new(tls_config))
```

Comment:
just qualify this Arc as `std::sync::Arc` and get rid of the conditional import up on line 58.

Reply:
Done! Thanks 🙂